File: spring-demos/spring-boot-actuator/src/main/java/nz/net/osnz/java/Application.java
package nz.net.osnz.java;

import org.springframework.boot.Banner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * @author Kefeng Deng (deng@51any.com)
 */
@SpringBootApplication
public class Application {

  public static void main(String[] args) {
    SpringApplication app = new SpringApplication(Application.class);
    app.setBannerMode(Banner.Mode.OFF);
    app.run(args);
  }

}

File: kotlin-demos/kotlin-sample-1/src/main/kotlin/nz/net/osnz/demo/Application.kt
package nz.net.osnz.demo

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.ResponseBody

/**
 * @author Kefeng Deng (deng@51any.com)
 */
@SpringBootApplication
@Controller
open class Application {

    @RequestMapping("/")
    @ResponseBody
    fun home(): String {
        return "hello world"
    }


}

fun main(args: Array<String>) {

    SpringApplication.run(Application::class.java, *args)


}